pkg/internal/metrics/metrics.go

```diff
@@ -79,6 +81,23 @@ var (
 		},
 		nil,
 	)
+
+	// crPolicyStatus tells you the status of the CertificateRequestPolicy's
+	// Ready condition. We don't report the `message` field from the condition
+	// because it isn't machine-readable (i.e., it's a free-text string that
+	// changes for each object and isn't meant to be used for filtering and
+	// grouping in Prometheus). We also don't report the `reason` field because
+	// it is paraphrasing the `status` (it says "Ready" or "NotReady").
+	crPolicyStatus = prometheus.NewDesc(
+		"approverpolicy_certificaterequestpolicy_status",
+		"Status of the CertificateRequestPolicy resources.",
+		[]string{
+			"namespace",
```

Comment:
Why do we include a namespace label? `CertificateRequestPolicy` is a cluster-scoped resource AFAIK.

Reply:
I am stupid. This namespace label is totally useless 😅 I’ll remove it.

Reply:
@maelvls Don't be so hard on yourself!

Reply:
Done!